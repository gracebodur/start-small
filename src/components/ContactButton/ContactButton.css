.cta-btn {
    margin-top: 25px;
    margin-bottom: 0 !important;
    background: var(--clr-blue);
    border-radius: 5px;
    border: none;
    color: var(--clr-light);
    padding: 15px 32px;
    cursor: pointer;
    position: relative;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
}

.cta-btn:hover {
    background-color: green;
    border: green;
}

.btn-inner {
    color: var(--clr-light);
    text-decoration: none;
    text-align: center;
    cursor: pointer;
    font-size: var(--fs-h3);
    text-transform: uppercase;
    font-family: var(--ff-primary);
    letter-spacing: var(--spacing-extra-small);
}

.btn-inner:hover {
    color: white !important;
    text-decoration: none;
}