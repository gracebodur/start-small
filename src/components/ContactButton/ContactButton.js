import React from 'react'
import './ContactButton.css'

function ContactButton() {
    return(
        <div >
            <button className='Btn center'>
                <a 
                href='mailto:startsmall-contact@gmail.com'
                target='_blank'
                rel='noopener noreferrer'
                >
                Contact Us
                </a>
            </button>
        </div>
    )
}

export default ContactButton