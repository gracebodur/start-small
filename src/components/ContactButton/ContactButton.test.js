import React from "react";
import toJson from "enzyme-to-json";
import ContactButton from "./ContactButton";

describe(`ContactButton component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a ContactButton by default", () => {
    const wrapper = <ContactButton />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the ContactButton given props", () => {
    const wrapper = <ContactButton {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
