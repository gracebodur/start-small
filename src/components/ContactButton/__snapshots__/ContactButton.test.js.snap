// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactButton component renders a ContactButton by default 1`] = `undefined`;

exports[`ContactButton component renders the ContactButton given props 1`] = `undefined`;
